cassandra/cluster.py

```diff
@@ -372,6 +373,14 @@ def auth_provider(self, value):
     If set to :const:`None`, there will be no timeout for these queries.
     """
 
+    idle_heartbeat_interval = 30
```

Comment:
I don't know if there has been discussion of the default value for this somewhere, but 30s feels a little high to me, especially for the control connection (since the driver may miss DOWN notifications while the control conn has failed).